app/models/user.rb

```diff
@@ -62,6 +64,14 @@ def get_visible_review_reqs_from(review_reqs)
     self.review_requests & review_reqs
   end
 
+  def check_and_return_image(type = nil)
+    if image.present?
+      type == :thumb ? image.thumb.url : image.url
```

Comment:
__[RuboCop]__

Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.


<!-- Comment by [Sider](https://sider.review) -->